Iguana2024/devops-2024
# Igd 17 develop bash script

## What type of PR is this? (check all applicable)


## Description
Added crontab.sh which use foreach to write logs to other VM's using ssh.

## QA Instructions, Screenshots, Recordings

To ensure that everything is working:

logging to your machine as a user vagrant
![image](https://github.com/Iguana2024/devops-2024/assets/64985383/d6e91bff-cc5e-489d-a236-adc5d356f399)
Use command "cat /var/tmp/sftp.log" to see list of logs from other machines
![image](https://github.com/Iguana2024/devops-2024/assets/64985383/8a199ca9-2394-4eac-88f8-f003e1d5295b)
If you have the same, then it works


## [optional] Are there any post deployment tasks we need to perform?
